inginious/frontend/webapp/templates/course_admin/edit_tabs/subproblem_templates.html

```diff
@@ -284,6 +284,363 @@ <h4 style="text-align: center">Choices</h4>
+            <div class="col-sm-10">
+                <input type="text" class="form-control" id="zoomStartScale-PID" name="problem[PID][options][zoom][startScale]" placeholder="Default is 1"/>
+            </div>
+        </div>
+        <div class="form-group">
+            <label for="zoomMaxScale-PID" class="col-sm-2 control-label">Max scale</label>
+            <div class="col-sm-10">
+                <input type="text" class="form-control" id="zoomMaxScale-PID" name="problem[PID][options][zoom][maxScale]" placeholder="Default is 3"/>
+            </div>
+        </div>
+        <div class="form-group">
+            <label for="zoomMinScale-PID" class="col-sm-2 control-label">Min scale</label>
+            <div class="col-sm-10">
+                <input type="text" class="form-control" id="zoomMinScale-PID" name="problem[PID][options][zoom][minScale]" placeholder="Default is 0.3"/>
+            </div>
+        </div>
+        <div class="form-group">
+            <label for="zoomScaleSpeed-PID" class="col-sm-2 control-label">Scale speed</label>
+            <div class="col-sm-10">
+                <input type="text" class="form-control" id="zoomScaleSpeed-PID" name="problem[PID][options][zoom][scaleSpeed]" placeholder="Default is 1.2"/>
+            </div>
+        </div>
+    </div>
+    <div id="subproblem_blockly_files">
+        <div class="form-group">
+            <label for="file-PID-INDEX" class="col-sm-2 control-label">File name</label>
+                <div class="col-sm-10">
+                    <input type="text" class="form-control" id="file-PID-INDEX" name="problem[PID][files][INDEX]" placeholder="Name of the file"/>
+                </div>
+        </div>
+    </div>
+    <div id="subproblem_blocks_files">
+        <div class="form-group">
+            <label for="file-PID-INDEX" class="col-sm-2 control-label">File name</label>
+                <div class="col-sm-10">
+                    <input type="text" class="form-control" id="file-PID-INDEX" name="problem[PID][blocks_files][INDEX]" placeholder="Name of the file"/>
+                </div>
+        </div>
+    </div>
+    <div id="subproblem_blockly_factory">
+        <div id="blocklyFactory"></div>
+
+        <!-- Blockly official JavaScript -->
+        <script src="$get_homepath()/static/common/js/blockly/blockly_compressed.js" type="text/javascript" charset="utf-8"></script>
+        <script src="$get_homepath()/static/common/js/blockly/blocks_compressed.js" type="text/javascript" charset="utf-8"></script>
+        <script src="$get_homepath()/static/common/js/blockly/javascript_compressed.js" type="text/javascript" charset="utf-8"></script>
+        <script src="$get_homepath()/static/common/js/blockly/python_compressed.js" type="text/javascript" charset="utf-8"></script>
+
+        <!-- Blockly French Translation -->
+        <script src="$get_homepath()/static/common/js/blockly/msg/js/fr.js" type="text/javascript" charset="utf-8"></script>
```

Comment:
Since the PR won't be merged anytime soon (we'll at least wait until June), let's wait until @anthonygego has merged the localisation system.